database/seeds/BuildMetricSeeder.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Illuminate\Database\Seeder;
+// composer require laracasts/testdummy
+use Laracasts\TestDummy\Factory as TestDummy;
+
+class BuildMetricSeederTableSeeder extends Seeder
```

Comment:
I would really like to re-work this to use regular seeding functionality if possible. We don't need exact data all the time imo.

Don't think we need to do it for this PR though.

Reply:
Yeah per my slack messages, just need an explanation on what you mean here. Main issue I was hitting is errors when running import normally for this many rows. Happy to implement a more standard solution, this is just the one I found that worked.

We can do for this PR, or at least document the appropriate method for a future PR.